site/gatsby-site/src/utils/Translator.js

```diff
@@ -83,9 +85,15 @@ class Translator {
       .db('translations')
       .collection(`incident_reports_${language}`);
 
-    const fields = [...keys, 'report_number'];
+    const translated = [];
 
-    const translated = items.map((item) => lodash.pick(item, fields));
+    for (const item of items) {
+      const { report_number, text, title } = item;
+
+      const plain_text = (await remark().use(remarkStrip).process(text)).contents.toString();
```

Comment:
Why not using the plain_text field here? Also, I thought translations used the markdown...

Reply:
That is storing the already translated text